mlir/test/Target/LLVMIR/openmp-teams.mlir

```diff
@@ -0,0 +1,136 @@
+// RUN: mlir-translate -mlir-to-llvmir -split-input-file %s | FileCheck %s
+
+llvm.func @foo()
+
+// CHECK-LABEL: @omp_teams_simple
+// CHECK: call void {{.*}} @__kmpc_fork_teams(ptr @{{.+}}, i32 0, ptr [[wrapperfn:.+]])
```

Comment:
Could you convert the captured variables (eg. wrapperfn) to caps? This is to distinguish them easily from code.